contracts/CO30_Munson_It's_the_amps_that_kill_you.VTF_Korsac/edit_me/briefing/objectives.html

```diff
@@ -0,0 +1,8 @@
+<font color='#E3D310'>PRIMARY</font>
+Recapture the Power Plant and Refinery.
+
+<font color='#595305'>SECONDARY</font>
+NONE
+
+<font color='#E3D310'>RESTRICTIONS</font>
+AVOID CIVILIAN CASUALTIES.
```

Comment:
```suggestion
<font color='#E3D310'>PRIMARY</font>
- Recapture the Power Plant
- Recapture the Refinery
```

They should always have `-` and be in the smallest form

We do not need secondary if there isn't any

We also must avoid civilian casualties, so it is not a mission restriction